cmd/api.go

```diff
@@ -163,41 +163,15 @@ func (o *apiOptions) validate(c *config.Config) error {
 
 	// In case we want to scaffold a resource API we need to do some checks
 	if o.doResource {
-		// Skip the following check for v1 as resources aren't tracked
-		if !c.IsV1() {
```

Comment:
We still needing keeping the V1/V2 checks besides the depreciation.

Reply:
The `v1` check is still done in [Config.HasResources()](https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/model/config/config.go#L73).

Reply:
Both checks are done in the method themselves (this way it will only need to be changed in one place when we remove v1).
- [`Config.HasResource()`](https://github.com/Adirio/kubebuilder/blob/1ac891650d2b641b4eb0b290ac07b93c998eb2a0/pkg/model/config/config.go#L61-L64)
- [`Config.ValidGroup()`](https://github.com/Adirio/kubebuilder/blob/1ac891650d2b641b4eb0b290ac07b93c998eb2a0/pkg/model/config/config.go#L98-L101)

Reply:
It is true. Its perfect!